core/src/main/java/io/questdb/ServerMain.java

```diff
@@ -158,10 +158,8 @@ public ServerMain(String[] args) throws Exception {
         log.advisoryW().$("available CPUs: ").$(Runtime.getRuntime().availableProcessors()).$();
         log.advisoryW().$("db root: ").$(cairoConfiguration.getRoot()).$();
         log.advisoryW().$("backup root: ").$(cairoConfiguration.getBackupRoot()).$();
```

Comment:
nit: we missed adding SQL copy root and work dir here and also `verifyFileSystem()` calls for them (when they're set). Might be a good idea to add them within this PR.

Reply:
@marregui is kindly refactoring ServerMain here: https://github.com/questdb/questdb/pull/2480

I feel that PR is better suited to tidy up logging, otherwise we would be magnifying merge conflicts.